bool beginDocking
bool moveToBin
float64 arucoX
float64 arucoY
